staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_docker.go

```diff
@@ -152,7 +156,9 @@ func NewCmdCreateSecretDockerRegistry(f cmdutil.Factory, ioStreams genericioopti
 	cmd.Flags().StringVar(&o.Email, "docker-email", o.Email, i18n.T("Email for Docker registry"))
 	cmd.Flags().StringVar(&o.Server, "docker-server", o.Server, i18n.T("Server location for Docker registry"))
 	cmd.Flags().BoolVar(&o.AppendHash, "append-hash", o.AppendHash, "Append a hash of the secret to its name.")
-	cmd.Flags().StringSliceVar(&o.FileSources, "from-file", o.FileSources, "Key files can be specified using their file path, in which case a default name will be given to them, or optionally with a name and file path, in which case the given name will be used.  Specifying a directory will iterate each named file in the directory that is a valid secret key.")
+	cmd.Flags().StringSliceVar(&o.FileSources, "from-file", o.FileSources, "Key files can be specified using their file path, "+
```

Comment:
I'm not quite sure this text is as helpful as it could be. I wonder if we can just keep the original help text and just insert `of .dockerconfigjson` in the text.  Something like:

> Key files can be specified using their file path, in which case a default name of .dockerconfigjson will be given to them, or optionally with a name and file path, in which case the given name will be used. Specifying a directory will iterate each named file in the directory that is a valid secret key.

Maybe we should even add something like this to the end:
> For this command, the key should always be .dockerconfigjson

It's probably a different conversation, but I wonder if we should actually be printing a warning if someone tries to specify a different key.

Am I correct in thinking that specifying any key other than .dockerconfigjson would create a secret, but just a regular one that would not be used by docker?  And if so, it would be a mistake if someone were doing that right?